source/configuration/write.txt

```diff
@@ -96,25 +96,15 @@ The following options for writing to MongoDB are available:
        | **Default:** ``_id``
 
    * - ``writeConcern.w``
-     - | Specifies ``w``, a write concern option to acknowledge the level to
-         which the change propagated in the MongoDB replica set. You can
-         specify one of the following values:
-
-       - ``MAJORITY``
-       - ``W1``, ``W2``, or ``W3``
-       - ``ACKNOWLEDGED``
-       - ``UNACKNOWLEDGED``
-
+     - | Specifies ``w``, a write-concern option to acknowledge the level to
+         which the change propagated in the MongoDB replica set. For a list
```

Comment:
[suggestion]

I had trouble parsing this sentence. I might suggest a rewording that I think makes it more clear. Also - does it actually acknowledge, or does it request an acknowledgement? 

```suggestion
     - | Specifies ``w``, a write-concern option to request an acknowledgement the write operation has
          propagated to a number of MongoDB instances in a replica set. For a list
```